src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  function add(n) {
+    calculator.result += n;
+  }
+
+  function subtract(n) {
+    calculator.result -= n;
+  }
+
+  function multiply(n) {
+    calculator.result *= n;
+  }
+
+  function divide(n) {
+    calculator.result /= n;
+  }
+
+  function reset() {
+    calculator.result = 0;
+
+    return this;
```

Comment:
[CODE KNOWLEDGE] For the `reset` method, use the method shorthand syntax to maintain consistency and enhance code readability.

BAD EXAMPLE:
 reset: function() {
   this.result = 0;
   return this;
 },

GOOD EXAMPLE:
 reset() {
   this.result = 0;
   return this;
 },